dist/PublicLab.Editor.js

```diff
@@ -22758,7 +22758,12 @@ module.exports = function CustomInsert(_module, wysiwyg) {
     }
   });
   const builder = require("./PublicLab.CustomInsert.Template.js");
-  $('.wk-commands').append('<a class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></a>');
+  $('.wk-commands').append('<button class="woofmark-command-insert btn btn-outline-secondary" data-toggle="insert" title="Custom Insert"><i class="fa fa-tags"></i></button>');
+
+  $(document).ready(function() {
+    $('[data-toggle="insert"]').tooltip();
+  });
+
```

Comment:
Hi @RaviAnand111, this change isn't a part of this PR.

Reply:
yeah while make the PR I was also confused about whats happening, then I check all the file on publiclab repo and there is something which I found,  `/src/modules/PublicLab.RichText.Custom-Insert.js` file and `/dist/PublicLab.Editor` are not same so on running `grunt build` `custom-insert` code is logged in `PublicLab.Editor.js` file, So how can I solve this

Reply:
You can yourself check these two files, the lines you pointed out are not same in these two files in official repo. 
@NARUDESIGNS @TildaDares